src/Domain/Regulation/Enum/RegulationOrderRecordSourceEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Regulation\Enum;
+
+enum RegulationOrderRecordSourceEnum: string
+{
+    public const DIALOG = 'dialog';
```

Comment:
```suggestion
    case DIALOG = 'dialog';
```
A utiliser de cette manière après : `RegulationOrderRecordSourceEnum::DIALOG->value`.